src/test/java/uk/gov/hmcts/reform/em/stitching/service/impl/DmStoreUploaderImplTest.java

```diff
@@ -55,37 +57,38 @@ private static Response intercept(Interceptor.Chain chain) {
             .build();
     }
 
-    @Test(expected = DocumentTaskProcessingException.class)
-    public void uploadFile() throws Exception {
+    @Test
+    void uploadFile() {
```

Comment:
Could we make the test name pertinent to what this will test? I.e. throwing a DocumentTaskProcessingException. E.g. testUploadFileThrowsDocumentTaskProcessingException?